### DISCUSSION TITLE
Disable report delete

### DISCUSSION THREAD

**Discussion by IvanKvesic** [INSPIRE-MIF/helpdesk-validator#1129] [Q&A] [answered]
Is there any option to disable other users to delete reports on validator?

**Comment by fabiovinci** [comment, marked as answer]
Dear @IvanKvesic,

currently, it is impossible to prevent other users from deleting reports because a login mechanism does not exist.

We recommend downloading the report in HTML format as well since they are automatically deleted after 48 hours.

**Reply by IvanKvesic** [reply]
Thank you very much for your quick reply, best regards
